tests/Controller/StatusControllerTest.php

```diff
@@ -25,4 +28,47 @@ public function testStatusFromElb()
         $this->assertResponseStatusCode($client, 200);
         $this->assertEquals('OK', $client->getResponse()->getContent());
     }
+
+    public function testNonConnectionDBErrorFromElb()
+    {
+        $client = static::createClient([], [
+            'HTTP_USER_AGENT' => 'ELB-HealthChecker/1.0',
+        ]);
+        // Create mock service to throw exception and inject into container
+        $mockService = $this->createMockProgrammesService();
+        $mockService->expects($this->once())
+            ->method('findByPidFull')
+            ->with(new Pid('b006m86d'))
+            ->willThrowException(new DBALException("Something bad happened."));
+        $client->getContainer()->set('pps.programmes_service', $mockService);
+
+        $crawler = $client->request('GET', '/status');
+
+        $this->assertResponseStatusCode($client, 500);
+        $this->assertEquals('ERROR', $client->getResponse()->getContent());
+    }
+
+    public function testConnectionDBErrorFromElb()
+    {
+        $client = static::createClient([], [
+            'HTTP_USER_AGENT' => 'ELB-HealthChecker/1.0',
+        ]);
+        // Create mock service to throw exception and inject into container
+        $mockService = $this->createMockProgrammesService();
+        $mockService->expects($this->once())
+            ->method('findByPidFull')
+            ->with(new Pid('b006m86d'))
+            ->willThrowException(new ConnectionException("Cannot Connect."));
+        $client->getContainer()->set('pps.programmes_service', $mockService);
+
+        $crawler = $client->request('GET', '/status');
+
+        $this->assertResponseStatusCode($client, 200);
+        $this->assertEquals('OK', $client->getResponse()->getContent());
+    }
+
+    private function createMockProgrammesService()
+    {
+        return $this->createMock('BBC\ProgrammesPagesService\Service\ProgrammesService');
```

Comment:
I believe `BBC\ProgrammesPagesService\Service\ProgrammesService::class` would be better as it is an actual class.  

- If removed, the IDE will be able to pick up the fact, otherwise it is a "meaningless" string.
- A developer would be able to click in to the class to see the methods etc...